ramenpos/RamenPos/View Controllers/Connection/ConnectionViewController.swift

```diff
@@ -102,12 +104,10 @@ class ConnectionViewController: UITableViewController, NotificationListener {
             return
         }
         
-        
-        RamenApp.current.client.acquirerCode = txtTenant.text != "Other" ? getTenantCode(tenantName: txtTenant.text!) : txtOtherTenant.text
+        RamenApp.current.client.tenantCode = txtTenant.text != "Other" ? getTenantCode(tenantName: txtTenant.text!) : txtOtherTenant.text
         RamenApp.current.settings.tenant = txtTenant.text != "Other" ? getTenantCode(tenantName: txtTenant.text!) : txtOtherTenant.text
```

Comment:
Sorry, I'm not familiar with how persistent storage works in iOS apps. What do these two lines do?